devfile.yaml

```diff
@@ -0,0 +1,69 @@
+# chectl devfile:generate --name=che-docs --git-repo=https://github.com/eclipse/che-docs --dockerimage=docker.io/antora/antora --command=./run_antora.sh
```

Comment:
Isn't it enough to have devfile.yaml? Since you also adds empty devfile.yml

Reply:
empty devfile.yml was a mistake ...